iterate_snowflake/README.md

```diff
@@ -8,28 +8,31 @@ First of all, you must connect your Tinybird workspace with your Git repository
 - Create a new git branch 
 - Create a new Snowflake .datasource data file in the branch, as in the example PR. You shouldn't need fixtures as the Data Source will be populated from the connection, but you can add fixtures and/or tests if needed.
   - For information about Snowflake options read the [connector documentation](https://www.tinybird.co/docs/ingest/snowflake)
-- Commit your code and create a new PR/MR from the branch to `main`
-- Wait for CI. You can check in the temporary workspace branch (automatically created and named like `tmp_ci_*`) that everything works as expected.
+- Commit your code and create a new PR/MR from the branch to `main` 
+- Wait for CI to finish.
+- Check that everything is working as expected in the temporary branch, CI will automatically create it and name it like `tmp_ci_{PRid}`).
 - Merge and wait for CD. Now you should have your Data Source in the main workspace, using the connection, and the temporary CI branch will be deleted.
 
-[Example Pull Request](https://github.com/tinybirdco/use-case-examples/pull/tbd)
+[Example Pull Request](https://github.com/tinybirdco/use-case-examples/pull/359)
 
 
 ### Update Snowflake Data Source with a new column, with same connection
 
 - Create another git branch
 - Add the new column to the schema and query
 - Commit your code and create an ew PR from the branch to `main`
-- Wait for CI to finish. You can check in the temporary workspace branch (automatically created and named like `tmp_ci_*`) that everything works as expected.
+- Wait for CI to finish.
+- Check that everything is working as expected in the temporary branch, CI will automatically create it and name it like `tmp_ci_{PRid}`).
 - Merge and wait for CD. Now you should have your Data Source updated in the main workspace, using the connection, and the temporary CI branch will be deleted.
 
-[Example Pull Request](https://github.com/tinybirdco/use-case-examples/pull/tbd)
+[Example Pull Request](https://github.com/tinybirdco/use-case-examples/pull/360)
 
 ### Delete a Snowflake Data Source
 
 - Create another git branch
 - Remove the .datasource data file
 - Commit your code and create an ew PR from the branch to `main`
-- Wait for CI to finish. You can check in the temporary workspace branch (automatically created and named like `tmp_ci_*`) that everything works as expected.
+- Wait for CI to finish.
+- Check that everything is working as expected in the temporary branch, CI will automatically create it and name it like `tmp_ci_{PRid}`).
 - Merge and wait for CD. Now you should have your Data Source updated in the main workspace, using the connection, and the temporary CI branch will be deleted.
```

Comment:
`Now you should have your Data Source updated in the main workspace`

The DS is deleted right? (only if it did not have dependencies I guess)

What about the connection is it still there?

Reply:
Added a note about dependencies an about the connection.